ignite/cmd/generate_vuex.go

```diff
@@ -12,7 +12,7 @@ import (
 func NewGenerateVuex() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "vuex",
-		Short: "Generate Typescript client and Vuex stores for your chain's frontend from your `config.yml` file",
+		Short: "*DEPRECATED* Generate Typescript client and Vuex stores for your chain's frontend from your `config.yml` file",
```

Comment:
You could also consider [Command.Deprecated](https://pkg.go.dev/github.com/spf13/cobra#Command.Deprecated), depending on the use case you want to have. The `Deprecated` property is useful when you want to keep the command but you don't want it to be visible. When the command is deprecated using this property it's hidden from the CLI help and when it's executed prints a deprecation message before anything else.

In a nutshell, `Command.Deprecated` is useful when the intention is to keep the command functional but hidden where its execution should print a deprecation message.